project/SqsDependencies.scala

```diff
@@ -0,0 +1,8 @@
+import sbt._
```

Comment:
We should use the latest version of aws sdk. https://github.com/zio/zio-connect/blob/master/project/AWSLambdaDependencies.scala

Reply:
I we should use zio-aws and build this connector on top of it. https://github.com/zio/zio-aws

Reply:
It's currently using the same versions as the S3 package. Maybe they should both be upgraded together? Or would you rather I update this one?